addons/mail/static/tests/call/call_tests.js

```diff
@@ -26,13 +26,15 @@ QUnit.test("basic rendering", async (assert) => {
     assert.containsOnce($, ".o-mail-CallParticipantCard[aria-label='Mitchell Admin']");
     assert.containsOnce($, ".o-mail-CallActionList");
     assert.containsOnce($, ".o-mail-CallMenu-buttonContent");
-    assert.containsN($, ".o-mail-CallActionList button", 7);
+    assert.containsN($, ".o-mail-CallActionList button", 5);
     assert.containsOnce($, "button[aria-label='Unmute'], button[aria-label='Mute']"); // FIXME depends on current browser permission
     assert.containsOnce($, ".o-mail-CallActionList button[aria-label='Deafen']");
-    assert.containsOnce($, ".o-mail-CallActionList button[aria-label='Raise hand']");
     assert.containsOnce($, ".o-mail-CallActionList button[aria-label='Turn camera on']");
-    assert.containsOnce($, ".o-mail-CallActionList button[aria-label='Share screen']");
-    assert.containsOnce($, ".o-mail-CallActionList button[aria-label='Enter Full Screen']");
```

Comment:
Assert `5` => immediately assert each item

It's always a good habit to link an asserted number to each item in following assertions.

In current diff, it feels off to see `5` => 4 assertions, followed by action, and then 4 assertions.